client/js/app.js

```diff
@@ -31,5 +31,5 @@ if (module.parent) {
     }
   }
 } else {
-  app.start('#app-root')
+  app.start('#app-root', {href: false})
```

Comment:
this is so navigating inside the archive doesn't update the browser history, right?